src/txn/txn_rollback_to_stable.c

```diff
@@ -531,14 +720,37 @@ static void
 __rollback_abort_newer_col_var(
   WT_SESSION_IMPL *session, WT_PAGE *page, wt_timestamp_t rollback_timestamp)
 {
+    WT_CELL *kcell;
+    WT_CELL_UNPACK_KV unpack;
     WT_COL *cip;
     WT_INSERT_HEAD *ins;
+    uint64_t recno, rle;
     uint32_t i;
+    bool stable_update_found;
+
+    if (page->dsk != NULL)
```

Comment:
Can you add a comment to explain the below code and how it works when there is no disk image?